content/accessible/use-semantic-html/index.md

```diff
@@ -16,7 +16,7 @@ By using the correct semantic HTML elements you may be able to meet most or all
 of your keyboard access needs. That means less time fiddling with `tabindex`,
 and more happy users!
 
-## Keyboard support for free (and better on mobile)
+## Keyboard support for free (and better mobile experiences)
```

Comment:
Changed this because I wasn't 100 percent sure what was meant by "better on mobile" (but it felt like the parenthetical was also an object of "for" in the prepositional phrase).  Let me know if this isn't correct!

Reply:
lgtm